.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "themes/book"]
+	path = themes/book
+	url = https://github.com/alex-shpak/hugo-book
```

Comment:
I think longer-term we'd want to eliminate the submodule dependency and write our own smaller, minimal theme that has similar style to r-l.o and does just what we need.